docs/processes/delivery/Continous/github-action/github-action-for-wordpress.md

```diff
@@ -0,0 +1,237 @@
+---
+id: GithubActions
+title: GitHub Actions for Wordpress
+sidebar_label: Wordpress GitHub Actions
+tags: [GitHub, Actions, CICD, Deployment, Wordpress, yml, WP Engine, AWS]
+---
+
+This document is an overview of the required information that a developer needs to have
+in order to implement CI/CD in any of the Wordpress projects.
+
+CI/CD automates your builds, and deployment so you can ship code changes faster and more reliably.
+
+# General Information
+
+### Platform
+The platform supported by the below scripts can run on GitHub since the below scripts are using GitHub action to achieve CI/CD.
+
+
+### Prerequisite
+- A GitHub account
+- Repository/Project on GitHub (public/private)
+- Necessary Permissions on the repository to store some SECRET KEYS in - Github (for more details connect with the IT team/Sagar)
```

Comment:
Not sure I understand this well enough. I don't see any harm in specifying what secrets.